network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -613,13 +665,26 @@ def exec(self):
                                        explanation)
         return QueryAnswer(True, self.name1 + ' and ' + self.name2 + ' are semantically equivalent.')
 
+    @staticmethod
+    def compute_query_output(query_answer, cmd_line_flag=False):
```

Comment:
```suggestion
    def compute_query_output(query_answer, _):
```
(`cmd_line_flag` is unused)